05_stringer/brnbke/main_test.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"bytes"
+	"strconv"
+	"testing"
+)
+
+func TestSortLetters(t *testing.T) {
```

Comment:
Name of testing function could be more descriptive for the actual function purpose

Reply:
Good point, perils of copying files from previous lab, corrected.